applications/nrf_desktop/doc/dfu.rst

```diff
@@ -149,6 +150,11 @@ sync
 
    Fetching the ``sync`` option also triggers moving the image data from the RAM buffer to flash.
 
+.. _dfu_bootloader_var:
+
+bootloader variant
```

Comment:
The option is actually named OPT_DESCR_MODULE_VARIANT ("module_variant") now. Please update.